src/print/index.ts

```diff
@@ -635,6 +635,8 @@ function expandNode(node): string {
         case 'Property':
             if (node.value.type === 'ObjectPattern') {
                 return ' ' + node.key.name + ':' + expandNode(node.value);
+            } else if (node.value.type === 'Identifier' && node.key.name !== node.value.name) {
+                return ' ' + node.key.name + ': ' + node.value.name;
```

Comment:
```suggestion
                return expandNode(node.key) + ':' + expandNode(node.value);
```

This is a minor nitpick, but I'd suggest using a recursive `expandNode` call instead. Partially for encapsulation reasons (there's only a single place in the code dealing with rendering identifiers), but also because e.g. `{:then { 2: a }}` seems to be syntactically valid (though it fails at runtime) and will format into `{:then { undefined: a }}`

Reply:
Done!